core/enumerable/uniq_spec.rb

```diff
@@ -0,0 +1,21 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+require File.expand_path('../fixtures/classes', __FILE__)
+
+ruby_version_is '2.4' do
+  describe 'Enumerable#uniq'do
+    before :each do
+      @enum = Object.new.to_enum
+      class << @enum
+        def each
+          yield 0, 'foo'
+          yield 1, 'FOO'
+          yield 2, 'bar'
+        end
+      end
+    end
+
+    it 'returns an array that contains only unique result of the given block' do
+      @enum.uniq { |_, label| label.downcase }.should == [0, 2]
+    end
```

Comment:
https://bugs.ruby-lang.org/issues/13669 ☑️ 

>It would be fine to keep that example, we can always change it later if Enumerable#uniq is changed.

👍 The example has been kept as  https://github.com/ruby/spec/pull/447/commits/5bc06e044f8be5804d7a12ce3dd05d160df21876.